src/components/oculus-touch-controls.js

```diff
@@ -254,6 +224,25 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     this.updateModel(buttonName, evtName);
   },
 
+  onAxisMoved: function (evt) {
+    var self = this;
+    var axesMapping = this.mapping[this.data.hand].axes;
+    // In theory, it might be better to use mapping from axis to control.
+    // In practice, it is not clear whether the additional overhead is worthwhile,
+    // and if we did grouping of axes, we really need de-duplication there.
+    Object.keys(axesMapping).forEach(function (key) {
+      var value = axesMapping[key];
+      var detail = {};
+      var changed = !evt.detail.changed;
+      if (!changed) { value.forEach(function (axisNumber) { changed |= evt.detail.changed[axisNumber]; }); }
```

Comment:
yes to the indentation, no to the else clause (because the forEach may change the value of `changed` and it needs to be re-evaluated)